### PR TITLE
TTVV Moriond18

### PR DESCRIPTION
**Pull request by jfernan2**
Not sure about settings at the end of the files run.card for systematics.

I see other MG5 LO cards do not have nothing after:

 True	=  use_syst

### CODE REVIEW

**Comment by efeyazgan** [comment]
please remove the banners from the proc cards (they show a different version)

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTHH_5f_LO/TTHH_5f_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTHH_5f_LO/TTHH_5f_LO_proc_card.dat]: please remove proton and jet definitions, and follow these instructions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by perrozzi** [comment]
apart from the comment on proton and jet definition, the rest looks good to me

**Comment by jfernan2** [comment]
These are 5f scheme samples. If I remove them, how can I guarantee b-quarks are considered as part of the proton/jet?

**Comment by perrozzi** [comment]
there is an automated check, cfr is5FlavorScheme in
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/gridpack_generation.sh#L353

**Pull request merged**